tests/async/test_async_model_methods.py

```diff
@@ -0,0 +1,33 @@
+from django.test import TestCase
+
+from .models import Book
+
+
+class AsyncModelOperationTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.s1 = Book.objects.create(
+            title="David Copperfield",
+        )
+        cls.s2 = Book.objects.create(
+            title="The Great Gatsby",
+        )
+
+    async def test_asave(self):
+        book = await Book.objects.aget(title="David Copperfield")
+        book.pages = 10
+        await book.asave()
+        getBook = await Book.objects.aget(title="David Copperfield")
+        self.assertEqual(book.pages, getBook.pages)
```

Comment:
Avoid refetching the instance by using its reference in `self`.

Avoid camelcase as per PEP 8.

```suggestion
        self.s1.field = 10
        await self.s1.asave()
        refetched = await SimpleModel.objects.aget()
        self.assertEqual(refetched.field, 10)
```